site/docs/src/pipeline.md

```diff
@@ -46,11 +46,11 @@ The pipeline has several stages:
 
 ### Modify pipeline configuration
 
-The `pipeline/config.yaml` file specifies how original `EO` programs are rewriteed into initial `EO` programs.
-Rewriteation is necessary because some objects in original `EO` programs don't work.
+The `pipeline/config.yaml` file specifies how original `EO` programs are rewrote into initial `EO` programs.
+Rewriting is necessary because some objects in original `EO` programs don't work.
```

Comment:
These changes should be removed too.

I suggest you:
- Undo the commits https://github.com/objectionary/normalizer/pull/369/commits/45064e68e9a1f18c57fa97fdc397de14066168ad, https://github.com/objectionary/normalizer/pull/369/commits/85a053b7fca3f7fba892c87a735e42d96570bc70
- Unstage changes in this file
- Commit again
- Force push. Force push is safe on a PR branch.